src/app/saga.ts

```diff
@@ -384,8 +386,9 @@ export function* handleSetAppState(action: SetAppState) {
 }
 
 export function* runCentralPhoneVerificationMigration() {
+  const odisV1EOL = yield select(odisV1EOLSelector)
   const shouldRunVerificationMigration = yield select(shouldRunVerificationMigrationSelector)
```

Comment:
nit: We should update `shouldRunVerificationMigrationSelector` instead and add the new condition there?
Otherwise this selector name becomes a bit misleading.

Reply:
ah oops, yes you're right. updated!